cmake/Modules/Packages/KIM.cmake

```diff
@@ -71,7 +71,11 @@ if(DOWNLOAD_KIM)
 else()
   if(KIM-API_FOUND AND KIM-API_VERSION VERSION_GREATER_EQUAL 2.2.0)
     # For kim-api >= 2.2.0
+    set_target_properties(lammps PROPERTIES INSTALL_RPATH_USE_LINK_PATH True)
+    set_target_properties(lmp    PROPERTIES INSTALL_RPATH_USE_LINK_PATH True)
     find_package(KIM-API 2.2.0 CONFIG REQUIRED)
+    get_target_property(_kim_loc KIM-API::kim-api IMPORT_LOCATION)        # Determine found message
```

Comment:
Isn't that stored in `KIM-API_DIR` as well?

Reply:
`KIM-API_DIR` will give the path to where the cmake config file is found, but IMPORT_LOCATION is the full path name of the actual library.  For example, I have a setup where `KIM-API_DIR` has the value `/Users/relliott/unison-sync/KIM/git/kim-api/build/test-install/share/cmake/kim-api` and `IMPORT_LOCATION` has the value `/Users/relliott/unison-sync/KIM/git/kim-api/build/test-install/lib/libkim-api.2.2.2.dylib`

I think using the import location is more consistent with the behavior of `find_package()` and `pkg_check_modules()`